zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -140,6 +140,35 @@ <h1>Edit upload</h1>
       <h1>New version</h1>
       {%- else %}
       <h1>New upload</h1>
+      {% if record.files %}
+      {%- if current_user_is_admin() %}
+      <div class="btn-group btn-block">
+        <button type="button" class="btn btn-m btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Admin <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu">
+          <li><a href="{{ url_for('persistentidentifier.index_view', flt1_21=record.id) }}">
+            Persistent identifiers
+          </a></li>
+          <li><a href="{{ url_for('recordmetadata.details_view', id=record.id) }}">
+            Record
+          </a></li>
+          <li><a href="{{ url_for('bucket.edit_view', id=record.files.bucket) }}">
+            Bucket
+          </a></li>
+          {%- endif %}
+          <li role="separator" class="divider"></li>
+          <li><a href="{{ url_for('zenodo_deposit.delete', pid_value=pid.pid_value) }}" type="submit">
+            <span class="text-danger"><i class="fa fa-trash"></i> Delete record ...</span>
+          </a></li>
```

Comment:
This entry doesn't make a lot of sense in the deposit... Unless you wrap it in an `if is_published` block

Reply:
You are right! Since it is not published it does not make sense! Going to remove it!

Reply:
You are right! Should I add the wrapping? Or should I remove the option?

Reply:
I think removing is fine. We usually delete records from the public record page (since we're sure it's published then)